src/main/frontend/config.cljs

```diff
@@ -338,6 +340,9 @@
 (def export-css-file "export.css")
 (def custom-js-file "custom.js")
 (def config-default-content (rc/inline "config.edn"))
+(def config-default-content-md5 (let [md5 (new crypt/Md5)]
+                                  (.update md5 (crypt/stringToUtf8ByteArray config-default-content))
+                                  (crypt/byteArrayToHex (.digest md5))))
```

Comment:
@RCmerci Have you tested the release mode? I'm not sure whether `.update` or `.digest` will be renamed by the closure compiler.

Reply:
haven't tested on release mode. but why `.update` will be renamed?

Reply:
Should be fine since crypt is from the goog closure library.